scripts/utils.sh

```diff
@@ -81,6 +84,11 @@ function setDockerComposeCLIOptions () {
             fi
         fi
         export dockerComposeFilesCLIOptions="$dockerComposeFilesCLIOptions -f ../$file"
+
+        # If running with Bahmni EMR, disable Demo service
+        if [ "$file" == *"-bahmniemr.yml" ]; then
+            export DEMO="false"
+        fi
```

Comment:
This logic doesn't work, I can still see demo service started. This is after starting Ozone with Bahmni using the start script `./start-demo.sh`

Reply:
Something could be wrong with your environment @corneliouzbett. Logic-wise, looks good. I've also gone an extra mile to re-run the setup and below is the finding...

<img width="1269" alt="Screenshot 2025-02-20 at 09 38 27" src="https://github.com/user-attachments/assets/7677658d-09a1-4e13-bed9-7ff9f8d91021" />

Reply:
Interesting, could be my environment

Reply:
I’ve added a more explicit reference to the file in question to see if it resolves the issue you're encountering. Could you try using the latest version and let me know, @corneliouzbett?